src/js/utils/DOM.js

```diff
@@ -64,18 +64,28 @@ export const containsFocus = node => {
   return !!element;
 };
 
-export const getFirstFocusableDescendant = element => {
+export const isFocusable = (element) => {
+  const tagName = element.tagName.toLowerCase();
+  return tagName === 'input' || tagName === 'select';
+};
+
+export const getFirstFocusableDescendant = (element) => {
   const children = element.getElementsByTagName('*');
   for (let i = 0; i < children.length; i += 1) {
     const child = children[i];
-    const tagName = child.tagName.toLowerCase();
-    if (tagName === 'input' || tagName === 'select') {
+    if (isFocusable(child)) {
       return child;
     }
   }
   return undefined;
 };
 
```

Comment:
Can you add a comment above this function explaining its use and basic logic? I'm not thinking something elaborate, just a sentence or two.